package.json

```diff
@@ -1,5 +1,10 @@
 {
-    "dependencies": {
-        "yuglify": "^0.1.4"
-    }
+  "dependencies": {
+    "yuglify": "^0.1.4"
+  },
+  "devDependencies": {
+    "gulp": "^3.9.1",
+    "gulp-jshint": "^2.0.0",
+    "jshint": "^2.9.1"
```

Comment:
Yeah, as of npm3, peer dependencies aren't installed anymore. Maybe they're targeting Node 4/5+ only.
On most of our other projects, we use ESLint. (completely unrelated topic, but I personally prefer ESLint).

See my comments below about possibly just removing all gulp code and just using JSHint (or ESLint) directly. Gulp is a pretty big hammer for for this use case. It's a _great_ hammer, but a bit big for a simple use case.